swarm/network/stream/stream.go

```diff
@@ -924,3 +929,13 @@ func (api *API) SubscribeStream(peerId enode.ID, s Stream, history *Range, prior
 func (api *API) UnsubscribeStream(peerId enode.ID, s Stream) error {
 	return api.streamer.Unsubscribe(peerId, s)
 }
+
+// hasStreamCap check if p2p.Peer the "stream" capability.
+func hasStreamCap(p *p2p.Peer) (yes bool) {
```

Comment:
Is it not better we put this function in `p2p/protocols` with an argument for protocol name?